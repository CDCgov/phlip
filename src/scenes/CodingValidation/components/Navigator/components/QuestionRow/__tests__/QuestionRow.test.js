import React from 'react'
import { shallow } from 'enzyme'
import { QuestionRow } from '../index'

const props = {
  item: {
    id: 2, text: 'question 3', indent: 0, number: '3', parentId: 0, positionInParent: 2, ancestorSiblings: [1],
    expanded: true,
    children: [
      { id: 3, text: 'question 4', indent: 1, number: '3.1', parentId: 2, positionInParent: 0 }
    ]
  },
  treeLength: 0,
  onQuestionSelected: () => {}
}

describe('Coding scene - QuestionRow', () => {
  test('should render correctly', () => {
    expect(shallow(<QuestionRow {...props} />)).toMatchSnapshot()
  })
})
