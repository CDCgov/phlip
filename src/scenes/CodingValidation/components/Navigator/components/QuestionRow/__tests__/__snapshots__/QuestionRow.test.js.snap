// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coding scene - QuestionRow should render correctly 1`] = `
<Fragment>
  <div
    className="undefined"
    key="2-scaffold-0"
    style={
      Object {
        "left": 0,
      }
    }
  />
  <div
    aria-label="Click to show this question"
    onClick={[Function]}
    role="row"
    style={
      Object {
        "alignItems": "center",
        "cursor": "pointer",
        "display": "flex",
        "height": 40,
        "marginLeft": 0,
        "outline": 0,
        "position": "relative",
      }
    }
  >
    <span
      style={
        Object {
          "maxHeight": 20,
          "maxWidth": 20,
          "minHeight": 20,
          "minWidth": 20,
          "outline": 0,
        }
      }
      tabIndex={-1}
    />
    <WithStyles(Typography)
      aria-label="Question number and text"
      noWrap={true}
      style={
        Object {
          "color": "white",
          "fontSize": 12,
          "fontWeight": 300,
          "outline": 0,
          "paddingLeft": 10,
          "paddingRight": 10,
        }
      }
      tabIndex={-1}
    >
      <span>
        3. 
      </span>
      question 3
    </WithStyles(Typography)>
  </div>
</Fragment>
`;
